content/learn/book/next-steps/_index.md

```diff
@@ -11,3 +11,5 @@ You have reached the end of The Bevy Book! And unfortunately, we haven't even sc
 * **[The Bevy Examples](https://github.com/bevyengine/bevy/tree/main/examples#examples)**: We create an example for every major Bevy feature. This is currently the best way to learn Bevy's features and how to use them.
 * **[Breakout](https://github.com/bevyengine/bevy/blob/main/examples/game/breakout.rs)**: A small "breakout" clone that illustrates what a real Bevy game looks like.
 * **[Bevy API Docs](https://docs.rs/bevy)**: Bevy's Rust API documentation.
+* **[Awesome Bevy](https://github.com/bevyengine/awesome-bevy)**: List of community projects in Bevy, including plugins, games, and learning resources.
```

Comment:
Double-listing community learning resources feels redundant. Maybe we should do something like:
```md
* **[Awesome Bevy](https://github.com/bevyengine/awesome-bevy)**: List of community Bevy projects, including [Learning Resources](https://github.com/bevyengine/awesome-bevy#learning)
, [Games](https://github.com/bevyengine/awesome-bevy#games), and [Plugins](https://github.com/bevyengine/awesome-bevy#plugins-and-crates)
```

Reply:
Or alternatively: (2) add "sub bullets" under awesome bevy (3) add top level bullets for each awesome-bevy category.

My preference is for the first option or (2)

Reply:
I like (2) the most; pushed a new commit to implement that.